### PR TITLE
Test installation of ArborX

### LINKED ISSUE #12: Test ArborX installation

### PR DESCRIPTION
**Pull request by dalg24**
Fix #12

### CODE REVIEW

File: .jenkins [hunk 1]
```diff
@@ -42,7 +47,7 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
-                        sh 'rm -rf build && mkdir -p build && cd build && cmake -D CMAKE_BUILD_TYPE=Debug -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=clang++ -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" -D MPIEXEC_PREFLAGS="--allow-run-as-root" .. && make -j8 VERBOSE=1 && ctest --no-compress-output -T Test'
+                        sh 'rm -rf build && mkdir -p build && cd build && cmake -D CMAKE_BUILD_TYPE=Debug -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=clang++ -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" -D ArborX_ENABLE_MPI=ON -D MPIEXEC_PREFLAGS="--allow-run-as-root" .. && make -j8 VERBOSE=1 && ctest --no-compress-output -T Test'
```
Comment (dalg24) [inline .jenkins]: Note the `$KOKKOS_DIR` listed in `CMAKE_PREFIX_PATH` :(

File: .jenkins [hunk 2]
```diff
@@ -21,13 +22,17 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
```
Comment (masterleinad) [inline .jenkins]: Do you still want to keep this?

File: .jenkins [hunk 3]
```diff
@@ -21,13 +22,17 @@ pipeline {
...
                             sh 'ccache --show-stats'
                             xunit([CTest(deleteOutputFiles: true, failIfNotNew: true, pattern: 'build/Testing/**/Test.xml', skipNoTestFiles: false, stopProcessingIfError: true)])
                         }
+                        success {
+                            sh 'cd build && make install'
+                            sh 'rm -rf tmp && mkdir -p tmp && cd tmp && cmake -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=$KOKKOS_DIR/bin/nvcc_wrapper -D CMAKE_PREFIX_PATH="$ARBORX_DIR;$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" ../examples/bvh_driver && make VERBOSE=1 && make test'
+                        }
                     }
                 }
                 stage('Clang') {
```
Comment (masterleinad) [inline .jenkins]: That is `CUDA-9.2`?

File: .jenkins [hunk 4]
```diff
@@ -21,13 +22,17 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
-                        sh 'rm -rf build && mkdir -p build && cd build && cmake -D CMAKE_BUILD_TYPE=Debug -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=$KOKKOS_DIR/bin/nvcc_wrapper -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" -D MPIEXEC_PREFLAGS="--allow-run-as-root" .. && make -j8 VERBOSE=1 && ctest --no-compress-output -T Test'
+                        sh 'rm -rf build && mkdir -p build && cd build && cmake -D CMAKE_INSTALL_PREFIX=$ARBORX_DIR -D CMAKE_BUILD_TYPE=Debug -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=$KOKKOS_DIR/bin/nvcc_wrapper -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" -D ArborX_ENABLE_MPI=ON -D MPIEXEC_PREFLAGS="--allow-run-as-root" .. && make -j8 VERBOSE=1 && ctest --no-compress-output -T Test'
                     }
                     post {
                         always {
                             sh 'ccache --show-stats'
```
Comment (masterleinad) [inline .jenkins]: ...and this?

File: .jenkins (hunk 2 above)
Comment (dalg24) [reply .jenkins]: Yes.  We only want the stats for that build.

Comment (dalg24) [reply .jenkins]: That said on a slave machine that allow concurrent builds the stat could be garbage

File: .jenkins (hunk 3 above)
Comment (dalg24) [reply .jenkins]: No GPU on that machine.  See the build argument `--build-arg BASE=ubuntu:18.04`

Comment (masterleinad) [reply .jenkins]: Ah, sure. I guess it's time to go home...

**Pull request merged**